src/data.json

```diff
@@ -11,662 +11,800 @@
+    "printtoconsole": {
+      "name": "Output to Console",
+      "description": "Send trace/debug info to console instead of debug.log.",
+      "default": false
     },
-    "cache_size_queue": {
-      "name": "Import Queue Cache",
-      "description": "Will keep up to {}MB of blocks in block import queue.",
-      "default": 50,
-      "min": 0,
-      "max": 32000
+    "promiscuousmempoolflags": {
+      "name": "Script Verification Flags",
+      "description": "Integer representing the script verification flags to enable (all ORed together) - flags can be found in interpreter.h.",
+      "default": ""
     },
-    "cache_size_state": {
-      "name": "State Cache",
-      "description": "Will keep up to {}MB data in State cache.",
-      "default": 25,
-      "min": 0,
-      "max": 32000
+    "shrinkdebugfile": {
+      "name": "Shrink Debug Log File",
+      "description": "Shrink debug.log file on client startup.",
+      "default": true
     },
-    "cache_size": {
-      "name": "Total Cache Size", 
-      "description": "If defined will never use more then {}MB for all caches. (Overrides other cache settings).",
-      "default": 0,
-      "min": 0,
-      "max": 32000
+    "testnet": {
+      "name": "Testnet",
+      "description": "Run this node on the Bitcoin Test Network.",
+      "default": false
     },
-    "fast_and_loose": {
-      "name": "Fast And Loose", 
-      "description": "Significant speed up but unclean exit is unrecoverable (disables DB WAL).",
+    "regtest": {
+      "name": "Regtest",
+      "description": "Run this node on its own independent test network.",
       "default": false
     }
   },
-  "snapshots": {
-    "section": "Snapshots",
-    "description": "Snapshoting (warp sync) settings.",
-    "disable_periodic": {
-      "name": "Disable Periodic",
-      "description": "Disables automatic periodic snapshots.",
+  "relay": {
+    "section": "Transaction Relay",
+    "description": "Fine tune settings for what transactions your node will relay to peers.",
+    "acceptnonstdtxn": {
+      "name": "Accept Non-Standard Transactions",
+      "description": "Relay and mine non-standard transactions (testnet/regtest only.)",
       "default": false
+    },
+    "incrementalrelayfee": {
+      "name": "Incremental Relay Fee",
+      "description": "Fee rate (in BTC/kB) used to define cost of relay, used for mempool limiting and BIP 125 replacement.",
+      "default": 0.00001,
+      "min": 0
+    },
+    "dustrelayfee": {
+      "name": "Dust Relay Fee",
+      "description": "Fee rate (in BTC/kB) used to defined dust, the value of an output such that it will cost about 1/3 of its value in fees at this fee rate to spend it.",
+      "default": 0.00001,
+      "min": 0
+    },
+    "bytespersigop": {
+      "name": "Bytes Per Signature Operation",
+      "description": "Equivalent bytes per sigop in transactions for relay and mining.",
+      "default": 20,
+      "min": 1
+    },
+    "datacarrier": {
```

Comment:
Yeah, they are the same. Though no one ever calls them "data carrier" transactions during normal usage.